### PR TITLE
burmese  localization (my) added

### PR DESCRIPTION
**Pull request by soewaiyanmyowin**
First of all , I really like using this app. I thought I should do more than just using the app. So , I decided to contribute to this project for my burmese fellas and for the sake of open-source community. Salute to all contributors and thank you.

### CODE REVIEW

**Comment by soewaiyanmyowin** [comment]
Hello! @kant  @JiLiZART @JesusM @Zorig can u pls review and merge?

File: SelfUpdate/my.lproj/Main.strings [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+"x3v-GG-iWU.title" = "Copy";
+
+/* Class = "NSMenuItem"; title = "Use All"; ObjectID = "xQD-1f-W4t"; */
+"xQD-1f-W4t.title" = "Use All";
+
+/* Class = "NSMenuItem"; title = "Speech"; ObjectID = "xrE-MZ-jX0"; */
+"xrE-MZ-jX0.title" = "Speech";
+
+/* Class = "NSMenuItem"; title = "Show Substitutions"; ObjectID = "z6F-FW-3nz"; */
+"z6F-FW-3nz.title" = "Show Substitutions";
```
Comment (jevonmao) [inline SelfUpdate/my.lproj/Main.strings:393]: Why is this file added? It does not seem necessary for the localization.

File: eul/my.lproj/Main.strings [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+"x3v-GG-iWU.title" = "Copy";
+
+/* Class = "NSMenuItem"; title = "Use All"; ObjectID = "xQD-1f-W4t"; */
+"xQD-1f-W4t.title" = "Use All";
+
+/* Class = "NSMenuItem"; title = "Speech"; ObjectID = "xrE-MZ-jX0"; */
+"xrE-MZ-jX0.title" = "Speech";
+
+/* Class = "NSMenuItem"; title = "Show Substitutions"; ObjectID = "z6F-FW-3nz"; */
+"z6F-FW-3nz.title" = "Show Substitutions";
```
Comment (jevonmao) [inline eul/my.lproj/Main.strings:393]: Same thing with this weird file, why is it added?

**Review by jevonmao — COMMENTED** [review]
Hey there! It's been a while since I last worked on this project, so maybe things have changed. The localization itself looks fine, but I'm not sure why Main.strings file was added?

File: eul/my.lproj/Main.strings (hunk 2 above)
Comment (soewaiyanmyowin) [reply eul/my.lproj/Main.strings:393]: These files are added by xcode automatically. Not from me.

Comment (gao-sun) [reply eul/my.lproj/Main.strings:393]: let me take care of this
